snap/snapcraft.yaml

```diff
@@ -112,39 +122,21 @@ parts:
         cp -r jars/* $CRAFT_PART_INSTALL/jars/
         mkdir -p $CRAFT_PART_INSTALL/python
         cp -r python/* $CRAFT_PART_INSTALL/python/
+
+        echo "Installing spark8t"
```

Comment:
**todo** I believe it is best (and recommended) to keep the different components separated in different `parts` ([ref](https://snapcraft.io/docs/adding-parts)). I'd install the `spark8t` package in a separate `part` as it was originally done before. I see, as the two (spark binaries and spark8t) are indeed two different components.

Reply:
If you preferer so.
Spark binaries are strongly bound to `spark8t`, so I don't think this would be a mess (one is incomplete w/o the other one in our context.) But if preferred I can split the 2 steps.

Reply:
Done